src/basicOp.js

```diff
@@ -24,7 +24,17 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  if (operation === '+') {
```

Comment:
It's better to use switch in this case. It makes code more understandable and takes less space.